_posts/2021-12-31-a-year-in-review-2021.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: epic
+title: "A Year in Review: 2021"
+date: 2021-12-31
+categories: [team, engineering]
+author: artsy_engineering
+---
+
+Wow - what a year! Here are some notable achievements, failures, and learnings from Artsy Engineering in 2021...
+
+<!-- more -->
+
+## Success!
+
+- Leveraged our investment in React Native to launch our Android App: Props to David, Mounir, Pavlos, Joanna, and the Collector Experience team
+- Moved X% of Force apps to V2 and Y% of Force traffic: Props to Chris, Damon, (...)
+- We killed X repos and retired N services
+- The awards for fastest PR from start date goes to: A! And even better, Ole did his first PR to Artsy 3 months before he joined.
+- We started and released N episodes of the Artsy Engineering Radio
+- We onboarded 23 engineers in Berlin - Props to Adam B, Christian and a whole team of willing mentors
+- Took down our poor CWV url numbers by 876786% (not verified)
+- Design system and Palette docs
+- Migrated Force and Eigen to new design system
+- Rebuilt the inquiry flow! Finally!
+- Migrated to a new ESP without incident (described as a project “i wouldn’t wish on my worst enemy”)
+- Hiring and investing a lot in turn-key set-up for new folks
+- New capabilities to our ecommerce functionality – ARTA, Avalara, Make Offer in Inquiries
+- Hired Luis, our first agile coach!
+- Collaborated with N iTechArt engineers on impactful projects
+- Moved a bunch of projects off of Heroku and onto our Kubernetes infrastructure
+- Spun up new collector-facing integrations: Algolia, Adjust, Braze, Split IO, Postie
+- Tech leads are taking more ownership over the success of their teams
+- Integrated our API with more 3rd party art management systems: ArtBase, Artsystems
+- Trialed integration with Split, experimentation and feature flagging vendor
+- Moved a bunch of pages in force to palette v3
+- More teams with focused goals
+- We opened 12,425 PRs
+- Responded to and resolved (N) incidents
+- After many false starts, we finally agreed to pull the trigger on a full bidding engine rewrite
+- More stable end-to-end automated testing (Integrity)
+- Worked with an external company for QA - Applause
+- Built a new data team from scratch
+- First new recommendation model built in a while - affinity score
+- Developed an in-house auction results pipeline
+- Made improvements in recommendations (New Works for You) for the first time in a while!
+- Migrated X repos to continuously deploy - Props to the Platform team
+- N questions answered in our #dev-help slack channel to help get each other unstuck!
+
+## Failures too!
+
+- We had X incidents
```

Comment:
I think this item can be covered at line _38 - Responded to and resolved (N) incidents_